subprojects/logging/src/main/java/org/gradle/util/SingleMessageLogger.java

```diff
@@ -42,9 +42,6 @@ protected Boolean initialValue() {
         }
     };
 
-    public static final String ORG_GRADLE_DEPRECATION_TRACE_PROPERTY_NAME = "org.gradle.deprecation.trace";
-    public static final String INCUBATION_MESSAGE = "%s is an incubating feature.";
```

Comment:
`INCUBATION_MESSAGE`?
Because it was only used by the local `incubatingFeatureUsed(String, String)` and `ParallelForkingGradleHandle` in integration tests.

It makes me feel safer if tests don't use "insider knowledge" like that constant.

If this constant is changed then I would prefer to see failing tests if they somehow depend on the value rather than tests that are still green but should have been reviewed.

And I inlined the variable because all other similar methods have their pattern defined inline.